src/style.css

```diff
@@ -1 +1,77 @@
-/* Styles go here */
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-style: normal;
+  --main-color: #00acdc;
+}
+  
+body {
+  padding: 0;
+  margin: 0;
+}
+  
+.header {
+  display: flex;
+  justify-content: space-between;
+  height: 60px;
+  padding: 0 50px;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+  
+.nav {
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+}
+  
+.nav__link {
+  list-style: none;
+}
+  
+.nav__list {
+  display: flex;
+  justify-content: flex-end;
+  padding: 0;
+  margin: 0;
+}
+  
+.nav__link:not(:last-child) {
+  margin-right: 20px;
+}
+  
+.nav__items {
+  font-family: inherit;
+  display: flex;
+  text-transform: uppercase;
+  text-decoration: none;
+  color: #000;
+  padding: 23px 0;
+  position: relative;
+}
+  
+.nav__items:hover {
+  color: var(--main-color);
+}
+  
+.is-active::after {
+  position: absolute;
+  content: "";
+  width: 37px;
```

Comment:
do not hardcode this, it should be reusable for any link

Reply:
haven't fixed, it was about width, it should be 100%